tests/e2e/tests/AggregatesSection/AggregatesHasDynamicTitle.spec.ts

```diff
@@ -34,7 +34,7 @@ test.beforeEach(async ({ page }) => {
   );
 });
 
-test(
+test.skip(
   qase(12, 'Aggregates title should be dynamic no-trigger'),
```

Comment:
@Piotrk39 I think this description can be a bit more detailed. The point of the test is that when you hover over the map, you see the title changing right? Can you include that a bit more?

Reply:
I changed it I hope it is clearer now. I also removed no-trigger part in the end since I believe it is true no matter the state.

Reply:
yes!